modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/components/filters/SelectionFilter.js

```diff
@@ -139,34 +335,71 @@ const SelectionFilter = ({
 			</ClayDropDown.Caption>
 			<ClayDropDown.Divider />
 			<ClayDropDown.Caption>
-				<div className="inline-scroller mb-n2 mx-n2 px-2">
-					{multiple ? (
-						<MultipleSelectionItems
-							items={items}
-							itemsValues={itemsValues}
-							onChange={(itemValue) => {
-								if (itemsValues.includes(itemValue)) {
-									setItemsValues(
-										itemsValues.filter(
-											(value) => value !== itemValue
-										)
-									);
-								}
-								else {
-									setItemsValues(
-										itemsValues.concat(itemValue)
-									);
-								}
-							}}
-						/>
+				<div className="form-group">
+					{items && !!items.length ? (
+						<ul
+							className="inline-scroller mb-n2 mx-n2 px-2"
+							ref={setScrollingArea}
+						>
```

Comment:
We are losing `ClayRadioGroup` here, but I guess it is worth it, for consistency and simplicity.